Lab_6.ipynb

```diff
@@ -0,0 +1,555 @@
+      "text/plain": [
+       "<Figure size 576x396 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "Pds = Psd.T\n",
+    "\n",
+    "index = 63\n",
+    "no_bins = array_size / signal_strength\n",
+    "val = index/no_bins\n",
+    "\n",
+    "x = np.linspace(dataedges[0], dataedges[-1], len(Pds[index]))\n",
+    "plt.step(x, Pds[index], where='mid')\n",
+    "\n",
+    "plt.ylabel(\"Probability\", fontsize=15)\n",
+    "plt.xlabel(\"Observed Signal, P(d|s)\", fontsize=15)\n",
+    "plt.title(\"Slice at True (injected) Value of \"+str(round(val)), fontsize=18)\n",
+    "plt.tick_params(labelsize = 13)\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "aa9d387b-cdcd-4148-9b31-e3ebf6902d36",
+   "metadata": {},
+   "source": [
+    "The signal has a lower probability but peaks at the same value (290)."
```

Comment:
Could maybe do an overlay with the known injection histogram to show what you mean by "lower probability"